instructions/humus-rest.instructions.md

```diff
@@ -0,0 +1,319 @@
+---
+description: 'Patterns and best practices for REST API applications using Humus'
+applyTo: '**/*.go'
+---
+
+# Humus Framework - REST Service Instructions
+
+This file provides patterns and best practices specific to REST API applications using Humus. Use this file alongside `humus-common.instructions.md` for complete guidance.
+
+## Project Structure
+
+### Simple Service (Single File)
```

Comment:
these instructions should be for full production quality applications so this section can be removed